blackbox/docs/sql/statements/explain.rst

```diff
@@ -25,16 +25,22 @@ The ``EXPLAIN`` command displays the execution plan that the planner generates
 for the supplied statement. The plan is returned as a nested object containing
 the plan tree.
 
+When issuing ``EXPLAIN ANALYZE`` the plan of the statement is also executed. In
+this case, timings of the different phases of the plan are returned instead of
+the plan itself.
```

Comment:
Maybe rephrase to: When issuing ``EXPLAIN ANALYZE`` the plan of the statement is executed and timings of the different phases of the plan are returned.